server/internal/oauth/impl.go

```diff
@@ -21,6 +21,7 @@ import (
 	goahttp "goa.design/goa/v3/http"
```

Comment:
Nice I'm really liking the way this file along with the providers package is working